### ISSUE TITLE
Make Gallery accessiblte

### ISSUE THREAD

**Issue by marcandre** [noelboss/featherlight#262]
Gallery should be accessible: previous & next should be `<button>`s, with `aria-label`s

**Comment by marcandre** [comment]
I'm stumped, the buttons and their inner spans get a width of 0? If I use an `<a>` instead, I don't have this problem.

Can anyone help? Checkout the `accessibility` branch and open `src/index.html` in your browser... @robneu? @mrwweb?

**Comment by robneu** [comment]
The only way I was able to override that was to either set an explicit width using a percentage or use the `-moz-available` prefix on Firefox, which probably isn't a very good idea.

I'm not sure why the button element is exhibiting that behavior when anchor tags aren't...

**Comment by robneu** [comment]
This fixes the issue, although I still have no idea what's causing it:

```css
.featherlight-next,
.featherlight-previous {
	display: block;
	position: absolute;
	top: 25px;
	right: 25px;
	bottom: 0;
	cursor: pointer;
	/* preventing text selection */
	-webkit-touch-callout: none;
	-webkit-user-select: none;
	-khtml-user-select: none;
	-moz-user-select: none;
	-ms-user-select: none;
	user-select: none;
	/* IE9 hack, otherwise navigation doesn't appear */
	background: rgba(0,0,0,0);
	border: none;
	padding: 0;
	width: 20%;
}

.featherlight-previous {
	left: 25px;
}
```

**Comment by marcandre** [comment]
Thanks for checking it out. Not exactly the same width than before, though 😢

**Comment by robneu** [comment]
Meh, seems close enough and that's a more typical way of styling something like that. I can't even really figure out how/why the other method works with other elements.
